benefits/templates/admin/login.html

```diff
@@ -4,19 +4,16 @@
   {{ block.super }} bg-secondary-subtle
 {% endblock bodyclass %}
 
-{% block dark-mode-vars %}
-{% endblock dark-mode-vars %}
-
```

Comment:
Instead of having `login`, `agency-base` all individually override `dark-mode-vars`, do it all in `base.html`.